### DISCUSSION TITLE
Feature Request: Show branch name when scrolling down in a PR

### DISCUSSION THREAD

**Discussion by akupila** [community/community#70286] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When viewing a PR, we can see the branch name on the top:
![image](https://github.com/community/community/assets/540683/a18a3fc5-bc76-4eac-b510-07eab0b3fd55)

However, when scrolling down, this info is hidden. I frequently want to check out a branch mid-review to view it locally, but the branch isn't easily accessible. As i don't want to lose the vertical scroll position in the review, i end up opening a new tab, only to get the branch name. There's plenty of space in the sticky header, so it would be nice to add the branch name there.

**Comment by jgcaruso** [comment]
100% this, but not just the branch name.

I'd like to see a bunch of more context in that scrolling header. Repo name would also be helpful as someone who works on many products across many repos, knowing which repo to apply that branch to would be really helpful!

**Comment by akupila** [comment]
The branch name is now shown (or has been for a while) when scrolling down so we can close this. Thank you!
